Automattic/wp-calypso
# Domains: migrate ICANN/email verification CSS to webpack

Migrates the styles of `IcannVerificationCard` to webpack and removes old unused stylesheet.

Removes a global override for `.notice__text` line wrapping, as it was 1) global, 2) never needed, as the success notice is inline. See below.

**How to test:**
In order to test, I patched the `domain.isPendingIcannVerification` property to be always `true`. Verify that when setting custom nameservers for a domain, a verification notice is shown:

<img width="727" alt="Screenshot 2019-03-26 at 13 15 14" src="https://user-images.githubusercontent.com/664258/54996349-5a172d80-4fc9-11e9-821e-ccb26d3a3334.png">

After resending the email, verify that an inline "notice" is shown. That makes the `.notice__text` style override redundant.
